usermods/battery_status_basic/usermod_v2_battery_status_basic.h

```diff
@@ -0,0 +1,260 @@
+#pragma once
+
+#include "wled.h"
+
+
+
+
+// pin defaults
+#ifndef TEMPERATURE_PIN
```

Comment:
:persevere: im so sorry... 
Since it is my first usermod oriented myself very strongly to the user mod "Temperature". 
As Aircookie had suggested it a 'long time' ago  #316